lib/taskbook.js

```diff
@@ -307,13 +307,16 @@ class Taskbook {
   checkTasks(ids) {
     ids = this._validateIDs(ids);
     const {_data} = this;
+    const [checked, unChecked] = [[], []];
 
     ids.forEach(id => {
       _data[id].isComplete = !_data[id].isComplete;
+      return _data[id].isComplete ? unChecked.push(id) : checked.push(id);
```

Comment:
Items whose `isComplete` attribute is updated to `true` should be placed in the `checked` array and items whose 
`isComplete` attribute is updated to `false` should be placed in the `unchecked` one:

```js
return _data[id].isComplete ? checked.push(id) : unchecked.push(id);
```

Reply:
No. I think it is right this way. The line above has already changed the status of the tasks

Reply:
`render.markIncomplete(checked);`
and then this

Reply:
The process you are describing is indeed identical, but these arrays, `checked[]` & `unchecked[]`, are meant to store item IDs based on each item's updated `isComplete` attribute value, and not on the original/database-stored one, thus reversing them will lead to a clearer and more obvious behavior for future readers. For example: an item stored in the database with its `isComplete` attribute set to `false` (unchecked), once selected & checked by the user, will have its value updated to its opposite `true` (checked), thus its ID should be passed to the `checked` array. The same holds for the opposite case, where the item is initial marked as complete.